extensions/amp-timeago/1.0/test/test-amp-timeago.js

```diff
@@ -15,14 +15,12 @@
  */
 import '../amp-timeago';
 import {toggleExperiment} from '../../../../src/experiments';
-import {waitForChildPromise} from '../../../../src/dom';
-import {whenCalled} from '../../../../testing/test-helper.js';
+import {waitFor} from '../../../../testing/test-helper.js';
 
 describes.realWin(
   'amp-timeago 1.0',
   {
     amp: {
```

Comment:
Why remove `runtimeOn`?

Reply:
It's much more deterministic to just call `await element.build()`.